runtime-display/xwayland/autobuild/beyond

```diff
@@ -0,0 +1,4 @@
+# xorg-server also provides these files
+abinfo "Removing duplicated files ..."
+rm -rvf "$PKGDIR/usr/lib/xorg"
```

Comment:
`-r` is not necessary here and `-f` should not be used. Also consider merging the two commands.

Reply:
Fixed.